h2o-r/h2o-package/R/modelselection.R

```diff
@@ -103,9 +111,13 @@
 #' @param custom_metric_func Reference to custom evaluation function, format: `language:keyName=funcName`
 #' @param nparallelism number of models to build in parallel.  Default to 0.0 which is adaptive to the system capability Defaults to
 #'        0.
-#' @param max_predictor_number Maximum number of predictors to be considered when building GLM models.  Defaiult to 1. Defaults to 1.
-#' @param mode Mode: used to choose model selection algorithm to use,  Must be one of: "allsubsets", "maxr". Defaults to
-#'        maxr.
+#' @param max_predictor_number Maximum number of predictors to be considered when building GLM models.  Default to 1. Defaults to 1.
+#' @param min_predictor_number For mode = 'backward' only.  Minimum number of predictors to be considered when building GLM models starting
+#'        with all predictors to be included.  Default to 1. Defaults to 1.
+#' @param mode Mode:  Used to choose model selection algorithms to use.  Options include 'allsubsets' for all subsets'maxr'
+#'        for MaxR'backward' for backward selection Must be one of: "allsubsets", "maxr", "backward". Defaults to maxr.
```

Comment:
```suggestion
#' @param mode Mode:  Used to choose model selection algorithms to use.  Options include 'allsubsets' for all subsets, 'maxr'
#'        for MaxR, 'backward' for backward selection Must be one of: "allsubsets", "maxr", "backward". Defaults to "maxr".
```

Note: Need to be fixed in gen_modelselection.py I guess.